src/odemis/acq/fastem.py

```diff
@@ -845,18 +844,39 @@ def estimateTiledAcquisitionTime(stream, stage, area, dwell_time=None):
     area_size = (acquisition_area[2] - acquisition_area[0],
                  acquisition_area[3] - acquisition_area[1])
     # number of tiles
-    nx = math.ceil(abs(area_size[0] / fov[0]))
-    ny = math.ceil(abs(area_size[1] / fov[1]))
+    nx = math.ceil(abs(area_size[0] / fov[0]))  # Number of tiles horizontally
+    ny = math.ceil(abs(area_size[1] / fov[1]))  # Number of tiles vertically
 
-    # add some overhead per tile for e.g. stage movement
-    overhead = 1  # [s]
-    # time per tile
+    # Time for tile acquisition
     dwell_time_value = dwell_time if dwell_time is not None else stream.emitter.dwellTime.value
-    acq_time_tile = res[0] * res[1] * dwell_time_value + overhead  # [s]
-    # time for total overview image
-    acq_time_overview = nx * ny * acq_time_tile  # [s]
+    acq_time_tile = res[0] * res[1] * dwell_time_value
 
-    return acq_time_overview
+    # Total acquisition time for imaging (all tiles)
+    overview_acq_time = nx * ny * acq_time_tile
+
+    # Stage movement time calculations
+    stage_speed_x = stage.speed.value['x']  # Speed of stage in x-direction [m/s]
+    stage_speed_y = stage.speed.value['y']  # Speed of stage in y-direction [m/s]
+
+    # Horizontal movement: Total time for moving across rows
+    time_x_per_row = (nx - 1) * (fov[0] / stage_speed_x)  # Moving (nx - 1) times per row
+    time_x = time_x_per_row * ny  # Repeated for each row
+
+    # Vertical movement: Time for repositioning to the next row
+    time_y_per_move = (fov[1] / stage_speed_y)  # Moving vertically between rows
+    time_y = time_y_per_move * (ny - 1)  # Moving (ny - 1) times
+
+    # Total stage movement time
+    stage_time = time_x + time_y
+
+    # Add overhead per tile
+    overhead_per_tile = 2  # [s]
```

Comment:
this feels like quite a lot, did you measure this? If you measured it, can you update the comment to include that that is how you ended up at 2 s?

Reply:
For `"resolution": (1024, 884)`, `"horizontalFoV": 2.0e-3,` `dwellTime: 100 ns`, `speed: 0.005, # m/s` below are the calculations:

`overview_acq_time = 5.07`
`stage_time = 21.62` 
`overhead = 112`

As you can see the image acquisition cannot be completed just by adding `overview_acq_time + stage_time`. I just found the `overhead_per_tile` number by doing an acquisition on the actual system which had the updated `PMD401Bus` driver.

Can I write the above as a comment?

Reply:
Also we reference the stage before starting an acquisition

Reply:
I think, I have a found a better fix, will discuss with you